privacyidea/lib/utils/__init__.py

```diff
@@ -1336,4 +1337,15 @@ def check_serial_valid(serial):
     """
     if not re.match(ALLOWED_SERIAL, serial):
         raise ParameterError("Invalid serial number. Must comply to {0!s}.".format(ALLOWED_SERIAL))
-    return True
\ No newline at end of file
+    return True
+
+
+def my_Popen(command, stdin=None, stdout=None, stderr=None, cwd=None, shell=False,
+             encoding=None):
```

Comment:
Could we pass `'utf8'` as the default encoding? This would save some code since it is the default...

Reply:
done